pages/instructorIssues/instructorIssues.ejs

```diff
@@ -2,12 +2,13 @@
 <html>
```

Comment:
Should we default to just showing open issues, like github does?

Reply:
Yeah, that's probably a good idea!